website_sale_credit_point/templates/navbar_points.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <template id="navbar_points" inherit_id="portal.frontend_layout">
+        <xpath expr="//ul[@id='top_menu']/*[1]" position="before">
```

Comment:
then why a wildcard? use `//ul[@id='top_menu']/li[1]`.
 `*` must be used only when you have no other chance

Reply:
this doesn't work. As I need it as a far left element. And the buttons like `Home`, etc. are not specified in the templates. If I specify it as `//ul[@id='top_menu']/li[1]` it will be before custom added buttons, like `Cart` but not the base one.

Reply:
@mpanarin I don't see how a dependency issue with `Cart` is solved that way. It should be the exact same result then no? I don't get why * would play differently, is it evaluated later?

Not sure if possible with templates, but could we not play with priority set to the lowest to ensure this one is added ?

Otherwise, maybe search for li with `<a href='/'>` I think we can assume that button `Home` will always be there.